NitroxClient/MonoBehaviours/Gui/HUD/RemotePlayerVitals.cs

```diff
@@ -132,6 +136,22 @@ private void CreatePlayerNameText()
             playerNameText.layer = em.gameObject.layer;
             // em does not have a parent anymore on latest stable, so the stats position is incorrect.
             playerNameText.transform.parent = em.transform;//.parent.transform;
+            GameObject signalBase = Instantiate(Resources.Load("VFX/xSignal")) as GameObject;
+            signalBase.name = "signal " + playerName;
+            signalBase.transform.position = Player.main.transform.position;
```

Comment:
Is it necessary to temporarily position it at the local player? Can't you leave it empty because it'll be overwritten in `SetBarPosition` soon anyway?

Reply:
I'm positioning signalBase here, and later I position the health bar to the position of the signal in SetBarPosition. I don't change the signalBase position in SetBarPosition

Reply:
Correct, I meant to edit that question to ask why you are setting it to the local player, beause this will cause the bars to float above the local player (as you're using it in `SetBarPosition`). I'd assume you want these to float above the other players instead.